package.json

```diff
@@ -12,7 +12,7 @@
     "router",
     "event router"
   ],
-  "main": "dist/esp.js",
+  "main": "index.js",
```

Comment:
Actually this one line isn't a reason for checking in the files. They shouldn't be checked in as you have suggested. My only sticking point is point package.json's main attributes to es6 JS files. 

Doing so means you need to transpile your dependencies and I don't think you should be doing that.